pkg/ddc/jindofsx/transform.go

```diff
@@ -659,15 +659,40 @@ func (e *JindoFSxEngine) transformFuse(runtime *datav1alpha1.JindoRuntime, value
 }
 
 func (e *JindoFSxEngine) transformLogConfig(runtime *datav1alpha1.JindoRuntime, value *Jindo) {
+	fsxProPerties := map[string]string{
+		"application.report.on":  "true",
+		"metric.report.on":       "true",
+		"logger.dir":             "/tmp/jindofsx-log",
+		"logger.cleanner.enable": "true",
+		"logger.consolelogger":   "true",
+		"logger.jnilogger":       "false",
+		"logger.sync":            "false",
+		"logger.verbose":         "0",
+	}
+
+	fuseProPreties := map[string]string{
```

Comment:
Do you mind renaming `fsxProPerties` to `fsxProperties`?  So does `fuseProPreties`?